src/commands/misc/tastedive.js

```diff
@@ -0,0 +1,41 @@
+const { SwitchbladeEmbed, Command} = require('../../')
+
+const MEDIA_WHITE_LIST = ['music', 'movies', 'shows', 'podcasts', 'books', 'authors', 'games']
+
+module.exports = class TasteDive extends Command {
+  constructor (client) {
+    super({
+      name: 'tastedive',
+      requirements: { apis: ['tastedive'] },
+      parameters: [
+        {
+          type: 'string',
+          full: false,
+          whitelist: MEDIA_WHITE_LIST,
+          missingError: 'commands:tastedive.noText' 
+        }, {
+          type: 'string',
+          full: true,
+          required: true,
+          missingError: 'commands:tastedive.noText' 
+        }
+      ]
+    }, client)
+  }
+
+  async run ({ channel, t, message }, media, term) {
+    const data = await this.client.apis.tastedive.search(media, term)
```

Comment:
We'd probably want to send an error message to the user if nothing is found

Reply:
I did that, but @Doges said that it's fine if I just log the error in the console

Reply:
I'll leave that up to him then

Reply:
> 
> 
> I did that, but @Doges said that it's fine if I just log the error in the console

Nah, I said that we have a CommandError system

Reply:
https://github.com/SwitchbladeBot/switchblade/blob/5b08e03088605d1756a807cbcee975083a87dec3/src/commands/games/minecraft/skin.js#L28

Reply:
So should I send an embed saying the error?

Reply:
No, the command error already does that, when it called it sends and error with the string, example
![image](https://user-images.githubusercontent.com/31023616/139595942-b05809bd-f550-46c6-84ad-419738450227.png)
https://github.com/SwitchbladeBot/switchblade/blob/5b08e03088605d1756a807cbcee975083a87dec3/src/commands/games/minecraft/server.js#L36

Reply:
Oh I see!